tests/selectedtests/task_mappings/test_mappings.py

```diff
@@ -0,0 +1,602 @@
+from unittest.mock import MagicMock, patch
```

Comment:
I think the filename for this file is a bit confusing (`test_mappings.py`), future engineers may think it's a misplaced test_mappings test file.

Could you rename the source file to `.../task_mappings/task_mappings.py` and rename this file to `test_task_mappings.py`?

(Yes, I realize the naming conventions of this project make things unnecessarily redundant to name.)

Reply:
Yup, naming in this project is hard. I've renamed the files as you've requested.